src/poliastro/coordinates.py

```diff
@@ -11,72 +11,285 @@
-    },
-    "Earth": {
-        "ra": 0.00 - 0.641 * T,
-        "dec": 90.00 - 0.557 * T,
-        "W": 190.147 + 360.9856235 * d
-    }
-}
-
-
-def body_centered_to_icrs(r, v, source_body, epoch=None, rotate_meridian=False):
+def mercury_rot_elements(epoch=J2000):
+    """Provides Mercury rotational elements given an epoch.
+
+    Provides north pole of body and angle to prime meridian
+
+    Parameters
+    ----------
+    epoch : ~astropy.time.Time, optional
+        Epoch, default to J2000.
+
+    Returns
+    -------
+    ra, dec, W: tuple (~astropy.units.Quantity)
+        Right ascension and declination of north pole, and angle of the prime meridian.
+
+    """
+
+    T = (epoch.tdb - J2000).to('day').value / 36525
+    d = (epoch.tdb - J2000).to('day').value
+
+    ra = (281.01 - 0.033 * T) * u.deg
+    dec = (61.45 - 0.005 * T) * u.deg
+    W = (329.548 + 6.1385025 * d) * u.deg
+
+    return ra, dec, W
+
+
+def venus_rot_elements(epoch=J2000):
+    """Provides Venus rotational elements given an epoch.
+
+    Provides north pole of body and angle to prime meridian
+
+    Parameters
+    ----------
+    epoch : ~astropy.time.Time, optional
+        Epoch, default to J2000.
+
+    Returns
+    -------
+    ra, dec, W: tuple (~astropy.units.Quantity)
+        Right ascension and declination of north pole, and angle of the prime meridian.
+
+    """
+
+    d = (epoch.tdb - J2000).to('day').value
+
+    ra = 272.76 * u.deg
+    dec = 67.16 * u.deg
+    W = (160.20 - 1.4813688 * d) * u.deg
+
+    return ra, dec, W
+
+
+def earth_rot_elements(epoch=J2000):
+    """Provides Earth rotational elements given an epoch.
+
+    Provides north pole of body and angle to prime meridian
+
+    Parameters
+    ----------
+    epoch : ~astropy.time.Time, optional
+        Epoch, default to J2000.
+
+    Returns
+    -------
+    ra, dec, W: tuple (~astropy.units.Quantity)
+        Right ascension and declination of north pole, and angle of the prime meridian.
+
+    """
+    T = (epoch.tdb - J2000).to('day').value / 36525
+    d = (epoch.tdb - J2000).to('day').value
+
+    ra = (0.00 - 0.641 * T) * u.deg
+    dec = (90.00 - 0.557 * T) * u.deg
+    W = (190.147 + 360.9856235 * d) * u.deg
+
+    return ra, dec, W
+
+
+def mars_rot_elements(epoch=J2000):
+    """Provides Mars rotational elements given an epoch.
+
+    Provides north pole of body and angle to prime meridian
+
+    Parameters
+    ----------
+    epoch : ~astropy.time.Time, optional
+        Epoch, default to J2000.
+
+    Returns
+    -------
+    ra, dec, W: tuple (~astropy.units.Quantity)
+        Right ascension and declination of north pole, and angle of the prime meridian.
+
+    """
+    T = (epoch.tdb - J2000).to('day').value / 36525
+    d = (epoch.tdb - J2000).to('day').value
+
+    ra = (317.68143 - 0.1061 * T) * u.deg
+    dec = (52.88650 - 0.0609 * T) * u.deg
+    W = (176.630 + 350.89198226 * d) * u.deg
+
+    return ra, dec, W
+
+
+def jupiter_rot_elements(epoch=J2000):
```

Comment:
Also, instead of having all these functions here and then the one below that maps body names to functions, perhaps we can have just methods in `Body`.

```
class Body:
    def __init__(self, parent, ...):
        # ...

    @time_magnitudes
    def rotational_elements(self, T, d):
        # ...
        return ra, dec, W
```

What do you think?